pkg/types/ovirt/machinepool.go

```diff
@@ -3,11 +3,66 @@ package ovirt
 // MachinePool stores the configuration for a machine pool installed
 // on ovirt.
 type MachinePool struct {
+	// InstanceTypeID defines the VM instance type and overrides
+	// the hardware parameters of the created VM, including cpu and memory.
+	// If InstanceTypeID is passed, all memory and cpu variables will be ignored.
+	InstanceTypeID string `json:"instanceTypeID,omitempty"`
```

Comment:
Can you mark these with +optional and kubebuilder default? See this for details https://github.com/openshift/installer/pull/3515/files#diff-ef5ad052319246b6e805b03c0221dcdcR34 and check out the whole PR for motivation.

Reply:
> If InstanceTypeID is passed, all memory and cpu variables will be ignored.
I think we need validation that user doesn't set these conflicting values in install-config.yaml